format/spec.md

```diff
@@ -456,9 +456,9 @@ The column metrics maps are used when filtering to select both data and delete f
 
 The manifest entry fields are used to keep track of the snapshot in which files were added or logically deleted. The `data_file` struct is nested inside of the manifest entry so that it can be easily passed to job planning without the manifest entry fields.
 
-When a file is added to the dataset, it’s manifest entry should store the snapshot ID in which the file was added and set status to 1 (added).
```

Comment:
The character `’` seems non-English character. It should be `its` instead of `it's` anyway so I fix this as a trivial change in this pull request.